src/Config.js

```diff
@@ -220,6 +224,14 @@ export class Config {
     }
   }
 
+  static validateMaxLimit(maxLimit) {
+    if (maxLimit) {
```

Comment:
Great catch - I've updated the tests and code accordingly.  lint also pointed out some spacing inconsistencies in files unrelated to my change so I updated those files too.